airbyte-integrations/connectors/source-smartsheets/acceptance-test-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+# Build latest connector image
+docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2)
```

Comment:
please add "-" after "-f2" and this script will parse a full image name (with tag value)